README.md

```diff
@@ -1 +1,33 @@
-# terraform-github-org-iam
\ No newline at end of file
+# terraform-github-org-iam
+
+This is the Terraform Github Module used to add users to the organization as members and admins, create teams and add users to their respective team.
+
+
+# Resources
+
+| Name | Type |
+|------|------|
+| [github_membership](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/membership) | resource |
+| [github_team](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/team) | resource |
+| [github_team_members](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/team_members) | resource |
+
+# Usage Example
+
+```terraform
+module "github-iam" {
+source = "github.com/coremaker/terraform-github-org-iam.git?ref=DOP-49-terraform-github-org-iam-Module"
```

Comment:
A little space here

Reply:
Maybe use source from root repo for example, without any ref/tagged version.